### PR TITLE
ZCS-1896 Added a warning regarding network modules ng ports usage and backup initialization

### PR DESCRIPTION
**Pull request by zextras-contrib**
only added a warning when network modules ng are enabled.

### CODE REVIEW

File: rpmconf/Install/Util/utilfunc.sh [hunk 1]
```diff
@@ -2381,6 +2381,23 @@ getInstallPackages() {
...
+      if [ $i = "zimbra-network-modules-ng" ]; then
+          echo "###WARNING###"
+          echo ""
+          echo "Network Modules NG needs to bind on TCP ports 8735 and 8736 in order"
+          echo "to operate, for inter-instance communication."
+          echo "Please verify no other service listens on these ports and that "
+          echo "ports 8735 and 8736 are properly filtered from public access "
+          echo "by your firewall."
+          echo ""
+          echo "Please remember that he Backup NG module needs to be initialized in order"
```
Comment (fatkudu) [inline rpmconf/Install/Util/utilfunc.sh]: s/he/the/

File: rpmconf/Install/Util/utilfunc.sh [hunk 2]
```diff
@@ -2381,6 +2381,23 @@ getInstallPackages() {
...
+          echo "to be functional. This is a one-time operation only that can be performed"
+          echo "by clicking the 'Initialize' button within the Backup section of the"
+          echo "Network NG Modules \in the Administration Console or by running"
```
Comment (fatkudu) [inline rpmconf/Install/Util/utilfunc.sh]: Why the "\"?

Comment (zextras-contrib) [inline rpmconf/Install/Util/utilfunc.sh]: typo

**Comment by fatkudu** [comment]
@zextras-contrib please sign the contributor agreement.  This needs to be signed once for each repository you contribute to.

**Comment by zextras-contrib** [comment]
"You have signed the CLA for Zimbra/zm-build" it was already signed hours ago :/

**Comment by zextras-contrib** [comment]
signed also as zextrasjay and it worked

**Comment by zextras-contrib** [comment]
the admin guide pull request has been created: https://github.com/Zimbra/adminguide/pull/13
do i need to change something else?

**Comment by gordyt** [comment]
@zextras-contrib There are requests for changes to the pull requests against the adminguide and installguides repos.  Please see the comments in the specific pull requests for more details.  Let me know if you need help.

**Comment by zextras-contrib** [comment]
admin guides were merged, jorge added the ports to the wiki

**Comment by zextras-contrib** [comment]
can this issue be merged?

**Comment by gordyt** [comment]
This pull request has been rebased and merged.

**Pull request closed without merging**